.vscode/settings.json

```diff
@@ -2,6 +2,6 @@
   "javascript.preferences.importModuleSpecifierEnding": "js",
   "typescript.preferences.importModuleSpecifierEnding": "js",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "typescript.tsdk": "node_modules\\typescript\\lib",
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
Could this impact Windows users?

Reply:
Often `/` works everywhere but `\` only works on Windows. If it does then we could likely remove this by then